File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDErrorLabel.m
//
//  PDErrorLabel.m
//  PDStrategy
//
//  Created by Pavel Deminov on 19/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

#import "PDErrorLabel.h"

@implementation PDErrorLabel

- (void)setup {
    [super setup];
    self.font = [UIFont systemFontOfSize:12];
    self.textColor = [UIColor redColor];
    self.numberOfLines = 0;
}

@end

File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDErrorLabel.swift
//
//  PDErrorLabel.swift
//  PDStrategy
//
//  Created by Pavel Deminov on 06/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

import UIKit

class PDErrorLabel: PDLabel {

    override func setup() {
        super.setup()
        
        font = UIFont.systemFont(ofSize: 12)
        textColor = UIColor.red
        numberOfLines = 0
    }

}
